Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -5,78 +5,123 @@
     var delegate: ReaderModeStyleViewControllerDelegate?
     var readerModeStyle: ReaderModeStyle = DefaultReaderModeStyle
 
-    private let rowHeights = [72, 44, 44, 52]
-    private let width = 260
-
-    private var rows: [UIView] = []
     private var fontTypeButtons: [FontTypeButton]!
+    private var fontSizeLabel: FontSizeLabel!
     private var fontSizeButtons: [FontSizeButton]!
     private var themeButtons: [ThemeButton]!
 
     override func viewDidLoad() {
-        view.backgroundColor = UIColor.grayColor()
-
         // Our preferred content size has a fixed width and height based on the rows + padding
 
-        let height = rowHeights.reduce(0, combine: +) + (rowHeights.count - 1)
-        preferredContentSize = CGSize(width: width, height: height)
-
-        // Setup the rows. It is easier to setup a layout like this when we organize the buttons into rows.
+        preferredContentSize = CGSize(width: ReaderModeStyleViewControllerUX.Width, height: ReaderModeStyleViewControllerUX.Height)
 
-        let fontTypeRow = UIView(), fontSizeRow = UIView(), themeRow = UIView(), sliderRow = UIView()
-        rows = [fontTypeRow, fontSizeRow, themeRow, sliderRow]
+        popoverPresentationController?.backgroundColor = ReaderModeStyleViewControllerUX.FontTypeRowBackground
 
-        for (idx, row) in enumerate(rows) {
-            view.addSubview(row)
+        // Font type row
 
-            // The only row with a full white background is the one with the slider. The others are the default
-            // clear color so that the gray dividers are visible between buttons in a row.
-            if row == sliderRow {
-                row.backgroundColor = UIColor.whiteColor()
-            }
+        let fontTypeRow = UIView()
+        view.addSubview(fontTypeRow)
+        fontTypeRow.backgroundColor = ReaderModeStyleViewControllerUX.FontTypeRowBackground
 
-            row.snp_makeConstraints { make in
-                make.left.equalTo(self.view.snp_left)
-                make.right.equalTo(self.view.snp_right)
-                if idx == 0 {
-                    make.top.equalTo(self.view.snp_top)
-                } else {
-                    make.top.equalTo(self.rows[idx - 1].snp_bottom).offset(1)
-                }
-                make.height.equalTo(self.rowHeights[idx])
-            }
+        fontTypeRow.snp_makeConstraints { (make) -> () in
+            make.top.equalTo(self.view)
+            make.left.right.equalTo(self.view)
+            make.height.equalTo(ReaderModeStyleViewControllerUX.RowHeight)
         }
 
-        // Setup the font type buttons
-
         fontTypeButtons = [
             FontTypeButton(fontType: ReaderModeFontType.SansSerif),
             FontTypeButton(fontType: ReaderModeFontType.Serif)
         ]
-        
+
         setupButtons(fontTypeButtons, inRow: fontTypeRow, action: "SELchangeFontType:")
 
-        // Setup the font size buttons
+        // Font size row
+
+        let fontSizeRow = UIView()
```

Comment:
We actually have a Toolbar class that takes a few buttons and spaces them out evenly inside it. Might be a little useful here (I guess that's what setupButtons does...). Follow up sometime if we've got repeated code there?